content/guides/10-minute-tutorial.md

```diff
@@ -104,6 +118,35 @@ Open the project in IntelliJ IDEA:
 
 {{% /block %}}
 
+{{% block "kotlin" %}}
+
+To use Kotlin in our project, we need to take some extra steps:
```

Comment:
While this sentence makes sense when viewing all languages together, there is no break for people who view it as Kotlin. The extra steps are apropos to nothing.

Reply:
Removed this sentence